tests/python_tests/ov_genai_test_utils.py

```diff
@@ -223,4 +222,5 @@ def load_pipe(configs: List[Tuple], temp_path):
 
 @functools.lru_cache(1)
 def get_continuous_batching(path):
-    return ov_genai.LLMPipeline(str(path), ov_genai.Tokenizer(str(path)), 'CB')
+    default_scheduler_config = ov_genai.SchedulerConfig
+    return ov_genai.LLMPipeline(str(path), ov_genai.Tokenizer(str(path)), 'CPU', config={"enable_continuous_batching": default_scheduler_config})
```

Comment:
do we have python bindings for properties of generation_config ? if yes, let's implement binding for enable_continuous_batching as well